src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -133,7 +134,9 @@ def _complex_params(command_group):
             c.argument('server_name', id_part=None, help='Name of the Server.')
 
         with self.argument_context('{} db'.format(command_group)) as c:
-            c.argument('database_name', options_list=['--name', '-n'])
+            c.argument('database_name', options_list=['--name', '-n'], id_part='child_name_1', help='The name of the database')
```

Comment:
Wondering if `id_part='child_name_1'` is necessary. I noticed that `database_name` for other commands has `id_part=None`. What's the difference between this command group and others?

Reply:
Thanks for pointing this out.. I realized for single server the command is 'az mysql db' and for flexible server the command is 'az mysql flexible-server db' so single server database name shouldn't have child_name_1 as ID part. so removed this

For flexible server, database name needs to be child name but I think we set the id part as none for every name parameters to avoid linter issue..!